DESCRIPTION

```diff
@@ -39,7 +39,8 @@ Authors@R: c(
   person("Michael","Quinn",        role="ctb"),
   person("@javrucebo","",          role="ctb"),
   person("@marc-outins","",        role="ctb"),
-  person("Roy","Storey",           role="ctb"))
+  person("Roy","Storey",           role="ctb"),
+  person("Manish","Saraswat",      role="ctb"))
```

Comment:
those entries should be managed by Matt and Arun only

Reply:
@jangorecki from Matt above:

> Also, please add a news item, and please add yourself to the bottom of contributors list in DESCRIPTION.

Reply:
resolved them